core/consensus/grandpa/grandpa_config.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_CONSENSUS_GRANDPA_GRANDPA_CONFIG_HPP
+#define KAGOME_CORE_CONSENSUS_GRANDPA_GRANDPA_CONFIG_HPP
+
+#include "consensus/grandpa/voter_set.hpp"
+
+namespace kagome::consensus::grandpa {
+
+  // Structure containing necessary information for running grandpa voting round
+  struct GrandpaConfig {
+    std::shared_ptr<VoterSet> voters;
+    RoundNumber round_number;
+    Duration duration;
+    Id peer_id;  // ed25519 public key of the peer, do not confuse with libp2p
```

Comment:
I would suggest to remove the mention of ed25519 here, as it breaks the abstraction Id provides (I mean, if Id is changed in the future to something different it would be extremely easy to forget to fix this comment and confuse things).